one_fm/one_fm/hr_utils.py

```diff
@@ -21,38 +27,96 @@ def indemnity_allocation_builder(employee_list):
         create_indemnity_allocation(employee)
 
 def create_indemnity_allocation(employee):
-    indemnity_allcn = frappe.new_doc('Indemnity Allocation')
-    indemnity_allcn.employee = employee.name
-    indemnity_allcn.from_date = employee.date_of_joining
-    indemnity_allcn.new_indemnity_allocated = 30/365
-    indemnity_allcn.total_indemnity_allocated = 30/365
```

Comment:
@SalShai @mymi14s This is not Indemnity amount, This is the Indemnity factor, No need to calculate the amount instead of this !!

Reply:
@pjamsheer According to the new requirement, Indemnity was calculated based on his no. of working years. if no. of working year is 5 or less, he gets 15 day's pay. above 5, he gets 30. And calculation involves taking per day payment(i.e. salary/26).